environments/operator/operator-config.libsonnet

```diff
@@ -31,14 +31,18 @@ cr.spec {
     version: thanosVersion,
     objectStorageConfig: objectStorageConfig,
   },
-  query: {
+  storeCache+:: {
+    image: cr.spec.storeCache.image,
+    version: cr.spec.storeCache.version,
```

Comment:
I am missing: 

- exporterVersion
- exporterImage
- replicas

Reply:
I don't want to make `exporter` configurable at this stage.

Reply:
OK, but you require it:
`exporterImage:: error 'must provide exporter image',`

Reply:
You can add it hardcoded in operator-config.libsonnet maybe...

Reply:
I've added exporter configuration now.